lte/gateway/c/core/oai/test/mock_tasks/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+package(default_visibility = ["//lte/gateway/c/core/oai/test:__subpackages__"])
+
+cc_library(
+    name = "mock_tasks",
+    srcs = [
+        "grpc_mock.cpp",
+        "ha_mock.cpp",
+        "mme_app_mock.cpp",
+        "s11_mock.cpp",
+        "s1ap_mock.cpp",
+        "s6a_mock.cpp",
+        "sctp_mock.cpp",
+        "service303_mock.cpp",
+        "sgs_mock.cpp",
+        "sgw_s8_mock.cpp",
+        "sms_orc8r_mock.cpp",
+        "spgw_mock.cpp",
+    ],
+    hdrs = [
+        "mock_tasks.h",
+    ],
+    deps = [
+        "//lte/gateway/c/core",
```

Comment:
Not sure why Bazel doesn't complain inside the devcontainer, but in the Magma VM I get a missing dependency error which can be resolved by adding "@com_google_googletest//:gtest_main",